.github/workflows/docker-image.yml

```diff
@@ -8,10 +8,25 @@ on:
 jobs:
 
   build:
+    strategy:
+      matrix:
+        arch: [ amd64, arm64 ]
+      fail-fast: false
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Build the Docker image
-      run: docker build . --file Dockerfile --tag whist-server:$(date +%s)
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          file: ./Dockerfile
+          platforms: linux/${{ matrix.arch }}
+          tags: |
+            whist-team/whist-server:${{ matrix.arch }}-${{ github.event.pull_request.head.sha }}
```

Comment:
will this actually work with normal pushes?

Reply:
Ah, good, I guess not. I'll have look later

Reply:
https://github.community/t/add-short-sha-to-github-context/16418/12 let's keep 'test'